static/src/javascripts/projects/commercial/modules/messenger/post-message.ts

```diff
@@ -1,9 +1,8 @@
 export const postMessage = (
 	message: Record<string, unknown>,
 	target: MessageEventSource,
-	targetOrigin?: string | null | undefined,
+	targetOrigin?: string,
```

Comment:
I think the idea here is to default to any origin:

```suggestion
	targetOrigin = '*',
```

Reply:
Misunderstood what you meant! 🤦